commit 30e052fa5c75692793a6bbd32f447f711be0b64a (refs/changes/04/5204/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-12-06T15:36:57-08:00 (10 months ago)
    
    MANTA-4030 many images using node 0.10 or 0.12 broken by transitive lru-cache@4.1.4 dep (pinning lru-cache version is no longer necessary after 4.1.5 release)

diff --git a/package.json b/package.json
index c3ec431..52b98ff 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "minnow",
     "description": "Agent that publishes disk stats to moray (for picking)",
-    "version": "1.0.1",
+    "version": "1.0.2",
     "author": "Joyent (joyent.com)",
     "license": "MPL-2.0",
     "private": true,
@@ -9,14 +9,12 @@
         "type": "git",
         "url": "git+ssh://git@github.com:joyent/manta-minnow.git"
     },
-    "// lru-cache": "attempting to pin version while satisfying transitive deps, because lru-cache@4.1.4 includes yallist@3 which breaks node --abort-on-uncaught-exception with node <v4",
     "dependencies": {
         "assert-plus": "0.1.4",
         "bunyan": "0.21.4",
         "cmdutil": "1.1.0",
         "dashdash": "1.3.2",
         "libmanta": "git+ssh://git@github.com:joyent/node-libmanta.git#master",
-        "lru-cache": "4.1.3",
         "moray": "3.3.0",
         "node-uuid": "1.4.0",
         "once": "1.2.0",
